.github/workflows/publish.yml

```diff
@@ -0,0 +1,38 @@
+name: Publish package to the RubyGems.org
+on:
+  release:
+    types: [published]
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'The version to release. v prefix is required (e.g. v1.10.0)'
+        required: true
+
+jobs:
+  push:
+    name: Push gem to RubyGems.org
+    runs-on: ubuntu-latest
+
+    permissions:
+      id-token: write # IMPORTANT: this permission is mandatory for trusted publishing
+      contents: write # IMPORTANT: this permission is required for `rake release` to push the release tag
+
+    steps:
+      - uses: actions/checkout@v4
+      - uses: ruby/setup-ruby@v1
+        with:
+          bundler-cache: true
+          ruby-version: 3.3
+      - name: Update version file with the release version
+        run: |
+          if [ "${{ github.event_name }}" == "workflow_dispatch" ]; then
+            VERSION=${{ github.event.inputs.version }}
+          else
+            VERSION=${{ github.event.release.tag_name }}
+          fi
+          VERSION=${VERSION#v}
+
+          sed -i "s/VERSION = \".*\"/VERSION = \"$VERSION\"/" lib/line/bot/api/version.rb
```

Comment:
This step is to avoid editing file to update version manually (https://github.com/line/line-bot-sdk-ruby/blob/fa36b30fdf2ef16a7724c755d56846681fd906c8/lib/line/bot/api/version.rb#L18)